src/client/App.jsx

```diff
@@ -71,6 +72,7 @@ export const App = () => {
                             <Route path={`/${user}`} element={<Homepage />} />
                             <Route path="/:user/:competitionId" element={<Competition />} />
                             <Route path="/:user/:competitionId/:seasonId" element={<Season />} />
+                            <Route path="/:user/:competitionId/create" element={<CreateSeason user={user} />} />
```

Comment:
you don't need to pass user as a prop anymore, I've used the Context (line 60) hook to provide it to any component that needs it. We could go for useReducer too, but lets see once we have a better idea of how many pieces of state we need.